core/whistle-1.0.0/src/wh_notify.erl

```diff
@@ -132,12 +134,8 @@ transaction(Account, Transaction, ServicePlan) ->
     wh_amqp_worker:cast(Notify, fun wapi_notifications:publish_transaction/1).
 
 -spec system_alert(atom() | string() | binary(), [term()]) -> 'ok'.
--spec system_alert(atom() | string() | binary(), [term()], wh_proplist()) -> 'ok'.
-
 system_alert(Format, Args) ->
-    system_alert(Format, Args, []).
-
-system_alert(Format, Args, Props) ->
+    Props = [],
```

Comment:
Ah, I see now, you're using detailed_alert/3 instead. Remove `<<"Details">>` from system_alert/2 then